libs/geometry/src/TangentSpaceMesh.cpp

```diff
@@ -87,21 +86,21 @@ std::string_view to_string(Algorithm algorithm) {
             return "FRISVAD";
         case Algorithm::FLAT_SHADING:
             return "FLAT_SHADING";
-        case Algorithm::SIGN_OF_W:
-            return "SIGN_OF_W";
         default:
             PANIC_POSTCONDITION("Unknown algorithm %u", static_cast<uint8_t>(algorithm));
```

Comment:
if a method is `noexcept` it shouldn't have any `PANIC_` in it. Because PANIC_ actually throws. In this case it's impossible to hit 'default' because you're switching on an enum. In fact it's better to not have the `default` because the compiler will complain if you don't have all the `case`

Reply:
done